src/wazuh_modules/syscollector/sysInfoWin.cpp

```diff
@@ -168,4 +463,48 @@ void SysInfo::getMemory(nlohmann::json& info) const
             "Error calling GlobalMemoryStatusEx"
         };
     }
-}
\ No newline at end of file
+}
+
+nlohmann::json SysInfo::getPackages() const
+{
+    return {};
+}
+
+nlohmann::json SysInfo::getProcessesInfo() const
+{
+    nlohmann::json jsProcessesList{};
+    PROCESSENTRY32 processEntry {};
+    processEntry.dwSize = sizeof(PROCESSENTRY32);
+    // Create a snapshot of all current processes
+    const auto spProcessSnapshot { CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS, 0) };
```

Comment:
use a unique_ptr with a deleter to handle the lifecycle of this snapshot handle.